rllib/connectors/common/batch_individual_items.py

```diff
@@ -196,3 +197,12 @@ def __call__(
             #    pass
 
         return batch
+
+    @override(Checkpointable)
+    def get_ctor_args_and_kwargs(self) -> Tuple[Tuple, Dict[str, Any]]:
+        return (
+            (),  # args,
+            {
+                "multi_agent": self._multi_agent,
```

Comment:
Okay so each connector must provide its arguments in this method to be correctly checkpointed. We might need to add this to the docs, as this might give users a bad experience, if they forget to add this in their own custom connectors and then run into problems loading their connector checkpoints.

Reply:
Yeah, great point. Let me try to provide some default implementation, which tries to gather all local variables from the call-frame below `super()` (the user's custom c'tor that calls `super()`) ...